tests/e2e/api/routing.js

```diff
@@ -537,7 +537,7 @@ describe('routing', () => {
         expect(results[2].statusCode).toEqual(403);
         expect(results[2].responseBody.error).toEqual('forbidden');
 
-        expect(results[3].statusCode).toBeFalsy();
+        expect(results[3].statusCode).toEqual(200);
```

Comment:
It fails because the `_security` object now contains the whole response object from the initial `_security` query performed during `add-national_admin-role` migration.

Reply:
Wow, good catch! Thanks!